src/test/tests/rendering/ospray.py

```diff
@@ -45,11 +47,108 @@ def TestOsprayWindowSave():
+    VolumeAtts.legendFlag = 1
+    VolumeAtts.lightingFlag = 0
+    VolumeAtts.opacityAttenuation = 1
+    VolumeAtts.opacityMode = VolumeAtts.FreeformMode
+    VolumeAtts.freeformOpacity = (255, 255, 255, 255, 255, 255, 255, 255, 255, 255,
+                                  255, 255, 255, 255, 255, 255, 255, 255, 255, 255,
+                                  255, 255, 255, 255, 255, 255, 255, 255, 255, 255,
+                                  255, 255, 255, 255, 255, 255, 255, 255, 255, 255,
+                                  255, 255, 255, 255, 255, 255, 255, 255, 255, 255,
+                                  255, 255, 255, 255, 255, 255, 255, 255, 255, 255,
+                                  255, 255, 255, 255, 255, 255, 255, 255, 255, 255,
+                                  255, 255, 255, 255, 255, 255, 255, 255, 255, 255,
+                                  255, 255, 255, 255, 255, 255, 255, 255, 255, 255,
+                                  255, 255, 255, 255, 255, 255, 255, 255, 255, 255,
+                                  255, 255, 255, 255, 255, 255, 255, 255, 255, 255,
+                                  255, 255, 255, 255, 255, 255, 255, 255, 255, 255,
+                                  255, 255, 255, 255, 255, 255, 255, 255, 255, 255,
+                                  255, 255, 255, 255, 255, 255, 255, 255, 255, 255,
+                                  255, 255, 255, 255, 255, 255, 255, 255, 255, 255,
+                                  255, 255, 255, 255, 255, 255, 255, 255, 255, 255,
+                                  255, 255, 255, 255, 255, 255, 255, 255, 255, 255,
+                                  255, 255, 255, 255, 255, 255, 255, 255, 255, 255,
+                                  255, 255, 255, 255, 255, 255, 255, 255, 255, 255,
+                                  255, 255, 255, 255, 255, 255, 255, 255, 255, 255,
+                                  255, 255, 255, 255, 255, 255, 255, 255, 255, 255,
+                                  255, 255, 255, 255, 255, 255, 255, 255, 255, 255,
+                                  255, 255, 255, 255, 255, 255, 255, 255, 255, 255,
+                                  255, 255, 255, 255, 255, 255, 255, 255, 255, 255,
+                                  255, 255, 255, 255, 255, 255, 255, 255, 255, 255,
+                                  255, 255, 255, 255, 255, 255)
+    VolumeAtts.samplesPerRay = 100
+    VolumeAtts.rendererType = VolumeAtts.RayCastingOSPRay
+    SetPlotOptions(VolumeAtts)
+    DrawPlots()
+
+    Test("ospray_domain_boundaries_00")
+
+    DeleteAllPlots()
+    CloseDatabase(data_path("vtk_test_data/poiseuille_rayleigh_benard_3D_00000600.visit"))
+
+
 def main():
     TestOsprayWindowSave()
+
+    #FIXME: this test always has a slight pixel diff (~0.01%). We
+    # need to add the ability to set pixel diff tolerance for
+    # individual tests before we can run this in our nightly.
+    # Currently, it's in the skip list.
+    TestOsprayVolumeDomainBoundaries()
```

Comment:
Even though we are skipping this test, probably best to also include the (best approximation) of the expected test baseline image too.